changelog.md

```diff
@@ -1,5 +1,9 @@
 # chartmogul-ruby Change Log
 
+## Version 4.0.0 - August 15 2023
```

Comment:
Do we really want it to be `4.0.0`? For me it feels more like `3.4.0`. It also should be updated in [version.rb](https://github.com/chartmogul/chartmogul-ruby/blob/main/lib/chartmogul/version.rb).